roles/requestrrx/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+#########################################################################
+# Title:            Community: RequestrrX                               #
+# Author(s):        Kalroth, Migz93, no5tyle, Mbryde                    #
+# URL:              https://github.com/Cloudbox/Community               #
+# Docker Image(s):  darkalfx/requestrr                                  #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Gather list of requestrrx Docker containers"
+  shell: "docker ps -a --format '{{ '{{' }} .Names{{ '}}' }}' --filter label=com.github.cloudbox.community=requestrrx | xargs echo -n"
+  register: cloudbox_community_sonarrx
```

Comment:
Could give it a new variable name here

Reply:
Should be fixed now in the newest commit